package.json

```diff
@@ -40,14 +40,15 @@
     "webpack": "^4.0.0"
   },
   "dependencies": {
-    "globby": "^7.1.1",
     "cacache": "^11.3.1",
     "find-cache-dir": "^2.0.0",
-    "serialize-javascript": "^1.4.0",
+    "globby": "^7.1.1",
     "is-glob": "^4.0.0",
     "loader-utils": "^1.1.0",
     "minimatch": "^3.0.4",
-    "p-limit": "^2.1.0"
+    "normalize-path": "^3.0.0",
+    "p-limit": "^2.1.0",
+    "serialize-javascript": "^1.4.0"
   },
   "devDependencies": {
     "@babel/cli": "^7.1.5",
```

Comment:
Don't touch this file in this PR

Reply:
is it okay that eslint warning `import/no-extraneous-dependencies`?

Reply:
@VdustR on what?

Reply:
https://github.com/webpack-contrib/copy-webpack-plugin/blob/6457c764334e3de36e86a5df8b88b9a52f24f6d3/src/preProcessPattern.js#L4

Reply:
Oh, all good, my mistake, tired